table.scss

```diff
@@ -0,0 +1,126 @@
+	display: table;
+	width: 100%;
+
+	& thead {
+		display: table-header-group;
+
+		& th {
+			@include vui-table-header-cell();
+		}
+	}
+
+	& tfoot {
+		& tr:first-child td {
+			border-top: 1px solid $vui-color-titanius;
+		}
+		& tr:first-child th {
+			border-top: 1px solid $vui-color-titanius;
+		}
+		display: table-footer-group;
+	}
+
+	& tr {
+		display: table-row;
+	}
+
+	& td {
+		@include vui-table-cell();
+	}
+
+	& tr td:last-child {
+		border-right: none;
+	}
+
+	& tr:last-child td {
+		border-bottom: none;
+	}
+
+	& tbody th,
+	  tfoot th {
+		@include vui-table-cell();
+	}
+
+	& tr th:last-child {
+		border-right: none;
+	}
+
+	//Correct for corners bleeding outside the rounded corners
+	& thead > tr:first-child > :first-child,
+	  tbody:only-child > tr:first-child > :first-child,
+	  tfoot:first-child + tbody:last-child > tr:first-child > :first-child,
+	  caption:first-child + tbody:last-child > tr:first-child > :first-child,
+	  caption:first-child + tfoot + tbody:last-child > tr:first-child > :first-child,
+	  caption:first-child + colgroup + tbody:last-child > tr:first-child > :first-child,
+	  caption:first-child + colgroup + tfoot + tbody:last-child > tr:first-child > :first-child,
+	  colgroup:first-child + tbody:last-child > tr:first-child > :first-child,
+	  colgroup:first-child + tfoot + tbody:last-child > tr:first-child > :first-child {
+		border-top-left-radius: 6px;
+	}
+	& thead > tr:first-child > :last-child,
+	  tbody:only-child > tr:first-child > :last-child,
+	  tfoot:first-child + tbody:last-child > tr:first-child > :last-child,
+	  caption:first-child + tbody:last-child > tr:first-child > :last-child,
+	  caption:first-child + tfoot + tbody:last-child > tr:first-child > :last-child,
+	  caption:first-child + colgroup + tbody:last-child > tr:first-child > :last-child,
+	  caption:first-child + colgroup + tfoot + tbody:last-child > tr:first-child > :last-child,
+	  colgroup:first-child + tbody:last-child > tr:first-child > :last-child,
+	  colgroup:first-child + tfoot + tbody:last-child > tr:first-child > :last-child {
+		border-top-right-radius: 6px;
+	}
+	& tfoot > tr:last-child > :first-child,
+	  tbody:only-child > tr:last-child > :first-child,
+	  thead + tbody:last-child > tr:last-child > :first-child,
+	  caption + tbody:last-child > tr:last-child > :first-child,
+	  colgroup + tbody:last-child > tr:last-child > :first-child {
+		border-bottom-left-radius: 6px;
+	}
+	& tfoot > tr:last-child > :last-child,
+	  tbody:only-child > tr:last-child > :last-child,
+	  thead + tbody:last-child > tr:last-child > :last-child,
+	  caption + tbody:last-child > tr:last-child > :last-child,
+	  colgroup + tbody:last-child > tr:last-child > :last-child {
+		border-bottom-right-radius: 6px;
+	}
+
+	& tfoot tr:last-child th,
+	  tbody:last-child>tr:last-child>th {
+			border-bottom: none;
+	}
+
+	//Apply hover to only rows that are not contained in thead, tfoot or have the class .vui-table-header or .vui-table-footer
```

Comment:
main use would be cases where thead was not used (technically thead is optional), though generally its old code that constructors tables that way,  so could push that down into the integration project for those cases.